src/components/forms/color/InputColorForm.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="input">
+    <select name="color" v-model="color" @change="updateStore">
+      <option value="natural">Pino natural</option>
+      <option value="red">Rojo</option>
+      <option value="blue">Azul</option>
+      <option value="yellow">Amarillo</option>
+      <option value="green">Verde</option>
+      <option value="pink">Rosado</option>
+      <option value="white">Blanco</option>
+      <option value="black">Negro</option>
+      <option value="grey">Gris</option>
+    </select>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+
+export default {
+  props: ['spot'],
+  data() {
+    return {
+      color: 'natural',
+    };
+  },
+  computed: {
+    ...mapGetters(['colorFurniture']),
+  },
+  methods: {
+    updateStore() {
+      this.colorFurniture[this.spot] = this.color;
```

Comment:
A tip: when you are accessing dynamically to somehting in an array, like ```javascript
const unTruc = unArray[unNumero]
```
you have to prepare your code for the case where this evaluates to `undefined` (because it's not in the array). 
```

So you can do stuff like:

```javascript
const unTruc = unArray[unNumero] ?? backUpValue
```

You don't always have to do something about it, but you have to THINK about that possibility and check if your code can lives with it